pydoctor/templatewriter/pages/__init__.py

```diff
@@ -137,9 +223,10 @@ def baseTables(self, request, tag):
         return ()
 
     def mainTable(self):
+        from pydoctor.templatewriter.pages.table import ChildTable
```

Comment:
This avoids a circular import because `ChildTable` inherits from `BaseElement`, right? Maybe `BaseElement` shouldn't be in `pydoctor.templatewriter.pages` then, but in for example `pydoctor.templatewriter` or in a new module.